src/WebJobs.Script.WebHost/Middleware/HostAvailabilityCheckMiddleware.Log.cs

```diff
@@ -14,13 +14,13 @@ private static class Logger
                 LoggerMessage.DefineScope<string>("Verifying host availability (Request = {RequestId})");
 
             private static readonly Action<ILogger, Exception> _initiatingHostAvailabilityCheck =
-               LoggerMessage.Define(LogLevel.Debug, new EventId(1, nameof(InitiatingHostAvailabilityCheck)), "Initiating host availability check.");
+               LoggerMessage.Define(LogLevel.Trace, new EventId(1, nameof(InitiatingHostAvailabilityCheck)), "Initiating host availability check.");
```

Comment:
We're writing way too many logs on every request (like 10+ logs per invocation). In high throughput scenarios, this is just too much. I've moved many of these to Trace statements, so they're not logged by default (default level for system logs is Debug). However, if the app is put into "diagnostic mode" the logs will be written.